src/script/page/RightSidebar/ConversationDetails/components/ConversationDetailsHeader/ConversationDetailsHeader.tsx

```diff
@@ -120,7 +125,14 @@ const ConversationDetailsHeader: FC<ConversationDetailsHeaderProps> = ({
             >
               {displayName && <span className="conversation-details__name">{displayName}</span>}
 
-              {canRenameGroup && <Icon.Edit className="conversation-details__name__edit-icon" />}
+              {canRenameGroup && (
+                <button
+                  className="conversation-details__name__edit-icon"
+                  aria-label={t('tooltipConversationDetailsRename')}
+                >
+                  <Icon.Edit aria-hidden="true" />
```

Comment:
do we need to pass aria-hidden? i think by default aria-hidden is true for icons?